src/hotspot/share/opto/intrinsicnode.hpp

```diff
@@ -198,6 +198,11 @@ class EncodeISOArrayNode: public Node {
   virtual uint ideal_reg() const { return Op_RegI; }
   virtual Node* Ideal(PhaseGVN* phase, bool can_reshape);
   virtual const Type* Value(PhaseGVN* phase) const;
+  virtual uint size_of() const { return sizeof(EncodeISOArrayNode); }
+  virtual uint hash() const { return Node::hash() + ascii; }
```

Comment:
Was like that before but since you're touching the code now, can you also add a leading `_` for the `ascii` field?

Reply:
Fixed